tests/unit_tests/test_analytics/test_sentry.py

```diff
@@ -0,0 +1,380 @@
+import socket
+import threading
+import time
+import zlib
+
+import flask
+import pytest
+import requests
+import wandb
+import wandb.analytics
+import wandb.env
+from flask import request
+from sentry_sdk.envelope import Envelope
+from sentry_sdk.utils import sentry_sdk
+
+EXTERNAL_SENTRY_DSN_KEY = "EXTERNALKEY"
+EXTERNAL_SENTRY_PROJECT = "123456"
+INTERNAL_SENTRY_DSN_KEY = "INTERNALKEY"
+INTERNAL_SENTRY_PROJECT = "654321"
+CLIENT_MESSAGE = "client message"
+WANDB_MESSAGE = "wandb message"
```

Comment:
i think it is better to just add it in the test, as it is just complicates reading